builder/entrypoint.sh

```diff
@@ -113,6 +118,17 @@ fi
 if [ "${1}" == "trace" ]; then
     TRACEE_EBPF_ONLY=1
     shift
+elif [ "${1}" == "custom-rules" ]; then
+    TRACEE_RULES="/custom-rules"
```

Comment:
don't u need to call "shift" to shift the arguments, if the user also wants to feed tracee-rules with arguments ?

Reply:
There's at 131.